src/backend/web/validation.js

```diff
@@ -43,11 +44,27 @@ function validateNewFeed() {
     const valid = ajv.validate(feedSchema, feedData);
 
     if (!valid) {
-      res.status(400).send();
+      return res.status(400).send();
     } else {
-      next();
+      return next();
     }
   };
 }
 
-module.exports.validateNewFeed = validateNewFeed;
+const validateParam = (req, res, next) => {
```

Comment:
Change the name to validatePostsIdParam please. It would reflect the purpose of your middleware better